cinema/migrations/0001_initial.py

```diff
@@ -77,6 +78,12 @@ class Migration(migrations.Migration):
                 ("title", models.CharField(max_length=255)),
                 ("description", models.TextField()),
                 ("duration", models.IntegerField()),
+                (
+                    "image",
+                    models.ImageField(
+                        null=True, upload_to=cinema.models.movie_image_path
```

Comment:
Ensure that `cinema.models.movie_image_path` is correctly defined in the `cinema.models` module. If it does not exist, you need to define it or correct the import path.